wright-filter-wheels/wright_filter_wheels/_yaqd_wright_filter_wheels_continuous.py

```diff
@@ -54,21 +59,21 @@ def direct_serial_write(self, message):
     def home(self):
         loop = asyncio.get_event_loop()
         loop.create_task(self._home())
+        self._set_position(self._position)
 
     async def _home(self):
         self._busy = True
-        # Initiate the home
         self._serial_port.write(f"H {self._motornum}\n".encode())
         await self._not_busy_sig.wait()
-        self._position=0
-
     
-    def set_microstep(self, microint):
+    def _set_microstep(self, microint):
         self._busy = True
         if microint in [2**i for i in range(0,6)]:
             self._serial_port.write(f"U {microint}\n".encode())
-            self.microstep=microint
-        
+            self._microstep=microint
+
+    def get_units(self):
```

Comment:
This is handled by the parent class, no need to explicitly include

Reply:
fixed as per above